neurom/fst/_mm.py

```diff
@@ -101,11 +101,6 @@ def map_sections(fun, nrn, neurite_type=NeuriteType.all):
                 for s in i_chain2(nrn.neurites, tree_filter=is_type(neurite_type)))
 
 
-def section_lengths(nrn, neurite_type=NeuriteType.all):
-    '''section lengths'''
-    return map_sections(mm.path_distance, nrn, neurite_type)
```

Comment:
Yes, this is irrelevant to tortuosity. I think the function you implemented is doing exactly what is expected. The discussion about the ```path_length``` / ```path_distance``` / ```section_length``` can be moved somewhere else (because this is still confusing, but has nothing to do with the actual functionality).